sapi/cli/tests/025.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+CLI php -i extension_dir
+--SKIPIF--
+<?php
+include "skipif.inc";
+if (substr(PHP_OS, 0, 3) == 'WIN') {
```

Comment:
Why not check `PHP_OS_FAMILY`?

Reply:
I just copied it from elsewhere, both ways are commonly used in tests. :shrug: